closed/adds/jdk/src/solaris/native/jdk/crypto/jniprovider/NativeCrypto_md.c

```diff
@@ -22,6 +22,7 @@
  * ===========================================================================
  */
 
+#include <link.h>
```

Comment:
This is only needed when `defined(UNIX)`; it may not exist on other platforms.

Reply:
I think in this case the solaris directory is just for unix.

Reply:
Keith mentioned solaris is used for aix and osx as well. It may or may not work.

Reply:
Actually, I don't see how `UNIX` would be defined. It's not predefined (at least not in gcc 7.5 on my Ubuntu system) and I don't see any makefiles that define it. So, `#if defined(UNIX)` yields dead code, including the list where we want to add `"libcrypto.so.3"`.

Reply:
Consistent with other comments, it should be:
```
#if defined(__linux__)
#include <link.h>
#endif /* defined(__linux__) */
```